poetry.lock

```diff
@@ -409,7 +424,7 @@ python-versions = "*"
 name = "isort"
 version = "5.10.1"
 description = "A Python utility / library to sort Python imports."
-category = "dev"
+category = "main"
```

Comment:
for example here I changed the dependency from dev to main by accident but don't know how to change it back except for deleting isort and installing it new... does anybody have a good idea on how to change it back? I can not do it manually as poetry computes a hash over that file